docs/frontend.md

```diff
@@ -0,0 +1,103 @@
+# Frontends
+
+A _frontend_ is a component of Chihaya that serves a BitTorrent tracker on one protocol.
+The frontend accepts, parses and sanitizes requests, passes them to the _backend_ (or _logic_) and writes responses to _Clients_.
```

Comment:
Let's not use the word backend because it doesn't exist anywhere in the code or docs.

The reality is we should decide whether the documentation is going to target users or developers.
If it's the former, we shouldn't bother talking about "Logic" because it's an interface that's implemented by our middleware. If it's the latter, we should decide what is the division between doc comments and mardown docs.